### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: src/Db/Dump/PKSimDB_dump.txt [hunk 1]
```diff
@@ -5955,9 +5955,9 @@ INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MetabolizationSpe
...
+INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveInfluxFromLumen_Hill','-TM * kcat * max(EffectiveLumenConcentration;0)^alpha / (Km^alpha+max(EffectiveLumenConcentration;0)^alpha)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveInfluxFromLumen_MM','-TM * kcat * EffectiveLumenConcentration / (Km+EffectiveLumenConcentration)','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaPgPToLumen_Hill','min(TM * kcat / (Km^alpha+EffectiveLumenConcentration^alpha); P_int_trans*Aeff*P_int_trans_lum_cell_factor) * EffectiveLumenConcentration^alpha + TM * kcat * K_water_cell * C_cell^alpha / (Km^alpha+K_water_cell * C_cell^alpha)','Amount per time');
+INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaPgPToLumen_Hill','min(TM * kcat * max(EffectiveLumenConcentration;0)^alpha / (Km^alpha+max(EffectiveLumenConcentration;0)^alpha); P_int_trans*Aeff*P_int_trans_lum_cell_factor * EffectiveLumenConcentration) + TM * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha+(K_water_cell * max(C_cell;0))^alpha)','Amount per time');
```
Comment (Yuri05) [inline src/Db/Dump/PKSimDB_dump.txt]: Replaced bit too much in those 2 formulas. Good this was covered by tests :)

**Comment by Yuri05** [comment]
@msevestre why are the tests not automatically performed in this PR?

**Comment by msevestre** [comment]
What do you mean not performed?

**Comment by msevestre** [comment]
Hum... Aren't they performed? Or is this just a GitHub issue?

**Comment by msevestre** [comment]
No idea. Weird

**Comment by Yuri05** [comment]
well, maybe temporarily issue with GitHub..

**Comment by msevestre** [comment]
[image: image.png]

Appveyor it seems

**Comment by msevestre** [comment]
https://help.appveyor.com/discussions/problems/23201-request-to-github-web-hook-has-been-made-from-prohibited-ip-address

**Pull request merged**